### PR TITLE
Added RecaptchaV2 task support

### PR DESCRIPTION
**Pull request by HugoLeblanc**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "anticaptcha",
-  "version": "2.0.3",
+  "name": "@hugoleblanc/anticaptcha",
```
Comment (ScreamZ) [inline package.json:2]: Please don't change package name

File: package.json [hunk 2]
```diff
@@ -24,9 +24,9 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/ScreamZ/anti-captcha.git"
+    "url": "https://github.com/HugoLeblanc/anti-captcha.git"
```
Comment (ScreamZ) [inline package.json:27]: Please don't change repository

File: package.json [hunk 3]
```diff
@@ -24,9 +24,9 @@
...
   },
   "bugs": {
-    "url": "https://github.com/ScreamZ/anti-captcha/issues"
+    "url": "https://github.com/HugoLeblanc/anti-captcha.git"
```
Comment (ScreamZ) [inline package.json:30]: Please don't change repository

**Review by ScreamZ — CHANGES_REQUESTED** [review]

**Comment by ScreamZ** [comment]
Thanks for the pull request, a few changes and we are ready to merge :-)

**Comment by HugoLeblanc** [comment]
Here we go, edited package.json.

**Review by ScreamZ — APPROVED** [review]

**Comment by ScreamZ** [comment]
Released under `2.0.6` on npm ✌🏻

**Pull request merged**